.github/actions/build-test/entrypoint.sh

```diff
@@ -2,14 +2,15 @@
 
 cd /github/workspace/source && mkdir build && cd build || exit 1
 
-echo "Configuring cmake"
+echo ">>> Configuring cmake"
 cmake -DBUILD_CONTROLLERS=ON -DBUILD_DYNAMICAL_SYSTEMS=ON -DBUILD_ROBOT_MODEL=ON -DBUILD_TESTING=ON ..
```

Comment:
I can imagine this line to fail if someone adds another lib but forget to add the dependencies in the image. Should you also put an `exit` here?

Reply:
True, I will briefly check if this step fails fatally by itself if I mess up the cmakelists. If it does I can leave it, but perhaps it would be nice to give it a custom log message and exit code as well.

Reply:
Updated to include configuration error check, tested with garbage CMakeLists and it worked